kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/PropertiesUi.java

```diff
@@ -628,10 +628,8 @@ private int findNextChannelId() {
 
             @Override
             public String getValue(final GwtChannelInfo object) {
-                final String value = object.get(id).toString();
-                return value == null ? "" : value;
+                return object.get(id).toString();
```

Comment:
if I try to execute this `final String value = object.get(id).toString();` and `object.get(id)` is null I would end in a NPE in any case when calling the `toString`.
Having `return value == null ? "" : value;` the following line would not prevent it.